core/chains/evm/log/pool.go

```diff
@@ -114,15 +144,19 @@ func (pool *logPool) deleteOlderLogs(keptDepth int64) *int64 {
 	return nil
 }
 
-func (pool *logPool) removeLog(log types.Log) {
+func (pool *logPool) RemoveBlock(blockHash common.Hash, blockNumber uint64) {
```

Comment:
nit/ the `block` prefix on the vars could be dropped since it's implied by the context.
```suggestion
func (pool *logPool) RemoveBlock(hash common.Hash, number uint64) {
```

Reply:
Thanks, done.